File: Problem Statements/Temperature Conversion/F - K/K2F.java

import java.util.Scanner;
class K2F{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Temperature (in Kelvin) : ");
        double k=sc.nextDouble();
        sc.close();
        double f=(k - 273.15) * 9/5 + 32;
        System.out.print("Temperature in Fahrenheit : "+f);
    }
}

File: Problem Statements/Temperature Conversion/F - K/K2F.c
#include<stdio.h>
void main(){
    double k,f=0.0;
    printf("Enter Temperature (in Kelvin) : ");
    scanf("%lf",&k);
    f=(k - 273.15) * 9/5 + 32 ;
    printf("Temperature in Fahrenheit : %0.2f",f);
}